### DISCUSSION TITLE
아바타의 손에 오브젝트를 들고 다니게 할수 있나요?

### DISCUSSION THREAD

**Discussion by HarmonyDH** [naverz/zepeto-studio-kor#785] [5. Q&A]
스크립트 작업에 초보입니다 ㅠ
아바타의 손에 오브젝트를 들고 다니게 할수 있나요?
예를들어 과자봉지 오브젝트 주변에 가면 버튼이 나타나고 버튼을 클릭하면 그 과자봉지가 아바타가 손에 들고다니게요
도움 부탁드립니다 ㅠㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 공식 샘플월드의 Chapter3 에 상호작용 및 인스턴스 생성 예제가 있습니다.
구현에 참고가 되면 좋겠습니다 😄
https://github.com/naverz/zepeto-studio-kor/discussions/615

**Reply by HarmonyDH** [reply]
감사합니다 참고해서 구현해보겠습니다 ㅠ

**Comment by yoonjoohwan** [comment]
상호작용 버튼이라 아니라 손의 위치를 찾는 것이라면 케릭터의 내부에 위치가 있어서 그 object 의 하위로 집어넣으면 될것같습니다 
무식한 방법이긴 하지만 ㅠㅠ
![image](https://user-images.githubusercontent.com/91707090/169940816-bda41090-2875-44a6-ad8d-db1fb6c334bc.png)

**Reply by HarmonyDH** [reply]
그런방법도 있군요 감사합니다

**Comment by Deytooh** [comment]
이렇게 코드 쓰면 돼요 ->

`myObject.transform.SetParent(MyZepetoPlayer.character.ZepetoAnimator.GetBoneTransform(HumanBodyBones.RightHand));`

:)

**Reply by HarmonyDH** [reply]
> 감사합니다 테스트 해볼게요 ㅠ
